data/curated/new_submission/meta.yaml

```diff
@@ -0,0 +1,15 @@
+title: "What have we been watching on Netflix?"
+article:
+  title: "What We Watched the First Half of 2025"
+  url: "https://about.netflix.com/en/news/what-we-watched-the-first-half-of-2025"
+data_source:
+  title: "Netflix Engagement Reports 2023-2025"
+  url: "https://assets.ctfassets.net/4cd45et68cgf/mplcXj5ulHDfbCPCr0f0I/5dbb6ec09f03df89706476e380e9b8bd/What_We_Watched_A_Netflix_Engagement_Report_2025Jan-Jun.xlsx"
```

Comment:
What are the other URLs? This can be a comma-separated list. A page that lists all the URLs would also work, if there is such a page.